examples/checkpanic/checkpanic.out

```diff
@@ -2,7 +2,4 @@
 # `.bal` file, and execute the `ballerina run` command.
 $ ballerina run checkpanic.bal
 120
-error: {ballerina}ConversionError {"message":"incompatible convert operation: 'string' value 'Invalid' cannot be converted as 'int'"}
-        at ballerina/utils:simpleValueConvert(<native>)
-           parse(checkpanic.bal:5)
-           main(checkpanic.bal:19)
+error: invoking main method failed due to error {ballerina}NumberParsingError message='string' value 'Invalid' cannot be converted to 'int'
```

Comment:
This again seems to be a bug (https://github.com/ballerina-platform/ballerina-lang/issues/17072), let's not change this.